contrib/auto-render/auto-render.js

```diff
@@ -102,9 +120,17 @@ const renderMathInElement = function(elem, options) {
         // treating it as if it were the start of a KaTeX math zone.
         {left: "\\[", right: "\\]", display: true},
     ];
-    optionsCopy.ignoredTags = optionsCopy.ignoredTags || [
-        "script", "noscript", "style", "textarea", "pre", "code",
-    ];
+
+    if (optionsCopy.ignoredTags && optionsCopy.shouldRender) {
+        throw new Error(
+            "Please specify either ignoredTags or shouldRender, not both"
```

Comment:
Why not both?

Reply:
I felt it was more intuitive to specify only `.ignoredTags` (`.shouldRender`) and have everything not classified as such implicitly satisfy `.shouldRender` (`.ignoredTags`).

Reply:
But the user may want to ignore some tags and other elements by another criterion.

Reply:
Would you provide an example of what you have in mind? The way I see it, if a user specifies both `optionsCopy.ignoredTags = ["script", "noscript", "style", "textarea", "pre", "code"]` and `optionsCopy.shouldRender = ["foo", "bar"]`, I wouldn't know what to do with a `<baz>` tag, whereas providing only one of the two properties would make it clearer.